flow/project.py

```diff
@@ -686,8 +690,8 @@ def __str__(self):
         return "{type}(op_func='{op_func}')" \
                "".format(type=type(self).__name__, op_func=self._op_func)
 
-    def __call__(self, job):
-        return self._op_func(job)
+    def __call__(self, jobs):
+        return self._op_func(*jobs)
```

Comment:
@vyasr @bdice Have we decided if we are just going to allow breaking changes to our API for aggregation? Technically `FlowOperation.__call__` is user facing. I don't mind since I am not sure of an easy way around this that is as clear.

Reply:
Good catch @b-butler. This can be made non-breaking if we change the argument from `jobs` to `*jobs` and made corresponding changes to the callers of this code.

Reply:
Yes, in general anywhere in this PR where we have to manually construct a tuple of jobs should be inspected to make sure it's not being caused by a missing a tuple unpacking in a function parameter. This is most important for user-facing APIs, but also good to do for internal APIs to ensure consistency and readability going forward.